server/store/kvstore/hashed_key.go

```diff
@@ -17,6 +18,8 @@ type hashedKeyStore struct {
 
 var _ KVStore = (*hashedKeyStore)(nil)
 
+const OAuth2KeyPrefix = "oauth2_"
```

Comment:
there should already be a constant for this

Reply:
Discussed